myAuthApp/routes/auth.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
+
+const User = require('../models/user');
+
+const bcryptSalt = 10;
+
+/* GET users listing. */
+
+router.get('/signup', (req, res, next) => {
+  if (req.session.currentUser) {
+    res.redirect('/main');
+    return;
+  }
+  res.render('../views/signup');
```

Comment:
handlebars is already looking inside the views folder because of this config in app.js: ```app.set('views', path.join(__dirname, 'views'));``` so no need to add it here